e2eRunner/src/main/scala/za/co/absa/hermes/e2eRunner/plugins/DatasetComparisonPlugin.scala

```diff
@@ -80,9 +80,11 @@ class DatasetComparisonPlugin extends Plugin with SparkBase {
     val args = testDefinition.args
     val testName = testDefinition.name
 
-    implicit val spark: SparkSession = sparkSession("DatasetComparisonPlugin")
```

Comment:
Tiny: We could revert this as well

Reply:
Reverted in commit 8a4f4502d565c4b5eba87a61115ac833713bd51c.